contracts/mocks/Assessment/ASMockClaims.sol

```diff
@@ -43,7 +42,7 @@ contract ASMockClaims is MasterAwareV2 {
       false // payoutRedeemed
     );
 
-    uint assessmentId = assessment().startAssessment(0, 0);
+    uint assessmentId = assessment().startAssessment(config.rewardRatio * requestedAmount / 10000, 0);
```

Comment:
why was it 0 before? just not implemented yet?

Reply:
I was not using the reward in this mock before.